enterprise_access/apps/bffs/constants.py

```diff
@@ -0,0 +1,25 @@
+"""
+Constants module for the BFFs.
+"""
+
+
+class COURSE_ENROLLMENT_STATUSES:
+    """
+    Course enrollment statuses.
+    """
+    IN_PROGRESS = 'in_progress'
+    UPCOMING = 'upcoming'
+    COMPLETED = 'completed'
+    SAVED_FOR_LATER = 'saved_for_later'
+    # Not a realized enrollment status, but used for the purpose of requesting enrollment
+    REQUESTED = 'requested'
+    # Not a realized enrollment status, but used for the purpose of assigned enrollment
+    ASSIGNED = 'assigned'
```

Comment:
Are we using these two anywhere?

Reply:
Currently, no. Carried over from [here](https://github.com/openedx/frontend-app-learner-portal-enterprise/blob/83e555a6909a0f0ce05023f6cf12994101ab0553/src/constants/course.js#L2-L11) in `frontend-app-learner-portal-enterprise`.

Eventually, the newly serialized `all_enrollments_by_status` would include `REQUESTED` and `ASSIGNED` to fully map to the corresponding data structure in the frontend. Opted to include the constants now, with the assumption they would be utilized down the road.